docs/targets/sendgrid.md

```diff
@@ -1,159 +1,71 @@
 
-## Creating a Sendgrid Target adapter. 
+![Adding a SendGrid target](../images/sendgrid-target/create-bridge-1.png)
 
-##### Once the Sendgrid Target Controller has been deployed, and all other needed assets are present, we can create integrations by adding SendgridTargets objects.
+In the Target creation form, provide a name for the event Target, and add the following information:
 
-1. Start by navigating to the [samples/](https://github.com/triggermesh/knative-targets/tree/Sendgrid-merge-fix/sendgrid/samples) folder, open the file named '100-secret.yaml', and update the 'apiKey' value with a valid Sendgrid API Key.
+* **Default sender name**: Define a default 'name' to be assigned in the `From:` section of the email to be created, if the received event does not contain a **FromName** property.
+* **Default sender email**: Define a default email address to be assigned in the `From:` section of the email to be created, if the received event does not contain a **FromEmail** property.
+* **Default recipient name**: Define a default name to be assigned in the `To:` section of the email to be created, if the received event does not contain a **FromEmail** property.
+* **Default recipient email**: Define a default 'email address' to be assigned in the `To:` section of the email to be created, if the received event does not contain a **ToEmail** property.
+* **API Secret**: Reference to a [TriggerMesh secret][tm-secret] containing an [API token][api] for authenticating requests
 
-2. Next move on to the '200-target.yaml' file in the same folder. Here was can imput some *optional* default values. If you do not require default values and want to pass these values dynamically via parameters in the CE body, you may omit some or all of these by deleteing the lines and leaving only the 'apiKey' portion.
+![SendgridTarget form](../images/sendgrid-target/create-bridge-2.png)
 
-    * A list of (optional) supported spec options: defaultFromEmail, defaultToEmail, defaultToName, defaultFromName, defaultMessage
+After clicking the `Save` button, the console will self-navigate to the Bridge editor. Proceed by adding the remaining components to the Bridge.
 
-3. Last file in the folder is ' 300-trigger.yaml' Here you can make modifications or leave it as is to subscribe to the default broker in the namespace you deploy. 
+After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge with a SendGrid event Target was successfully created.
 
+![Bridge status](../images/bridge-status-green.png)
 
-### Status
+For more information about using SendGrid, please refer to the [SendGrid documentation][docs].
 
-* The SendgridTarget requires only one Secret, the APIKey, to be provided. 
+### Event Type
 
-* Once its presences is confirmed, the Sendgrid Target Controller will create a Knative Service for the Adapter.  
+The SendGrid event target can consume events of any type[^s]
```

Comment:
This funny character at the end does not render, check the rendered page.

Reply:
that is odd.. it renders in my markdown editor but i guess GitHub doesn't like it for some reason.

Reply:
oh.. looks like it is valid markdown but GitHub has an issue with footnotes. Ref-> https://github.com/github/markup/issues/498

Reply:
fixing now :)